src/lib/kdb/kdb_convert.c

```diff
@@ -411,20 +411,7 @@ ulog_conv_2logentry(krb5_context context, krb5_db_entry *entry,
             break;
 
         case AT_LAST_FAILED:
-            if (!exclude_nra && entry->last_failed >= 0) {
-                ULOG_ENTRY_TYPE(update, ++final).av_type = AT_LAST_FAILED;
-                ULOG_ENTRY(update, final).av_last_failed =
-                    (uint32_t)entry->last_failed;
-            }
-            break;
-
         case AT_FAIL_AUTH_COUNT:
-            if (!exclude_nra) {
-                ULOG_ENTRY_TYPE(update, ++final).av_type =
-                    AT_FAIL_AUTH_COUNT;
-                ULOG_ENTRY(update, final).av_fail_auth_count =
-                    (uint32_t)entry->fail_auth_count;
-            }
```

Comment:
Here we're removing some speculative code, based on the comment "always exclude non-replicated attributes, for now".

This is probably fine; I don't see any circumstance where we'd want to transmit these fields over iprop.  But at the same time we should hit AT_LAST_SUCCESS, remove the case labels (there's already a default-break), and remove the exclude_nra variable and just pass TRUE to find_changed_attrs().